### PR TITLE
[#5270] UI(oceanbase-catalog): Support OceanBase-CE JDBC catalog on the Web UI

### PR DESCRIPTION
**Pull request by LauraXia123**
### What changes were proposed in this pull request?
Support OceanBase-CE JDBC catalog on the Web UI
image

### Why are the changes needed?
N/A

Fix: #5270

### Does this PR introduce _any_ user-facing change?
N/A

### How was this patch tested?
N/A

### CODE REVIEW

File: web/web/src/lib/utils/initial.js [hunk 1]
```diff
@@ -308,5 +308,33 @@ export const providers = [
...
+        key: 'jdbc-driver',
+        value: '',
+        required: true,
+        description: 'e.g. com.mysql.jdbc.Driver or com.mysql.cj.jdbc.Driver'
+      },
+      {
+        key: 'jdbc-url',
+        value: '',
+        required: true,
+        description: 'e.g. jdbc:mysql://localhost:3306'
```
Comment (yuanoOo) [inline web/web/src/lib/utils/initial.js:326]: The default sql port of OceanBase is 2881.

Comment (LauraXia123) [reply web/web/src/lib/utils/initial.js]: fixed

File: web/web/src/lib/utils/initial.js [hunk 2]
```diff
@@ -308,5 +308,33 @@ export const providers = [
...
+  },
+  {
+    label: 'OceanBase',
+    value: 'jdbc-oceanbase',
+    defaultProps: [
+      {
+        key: 'jdbc-driver',
+        value: '',
+        required: true,
+        description: 'e.g. com.mysql.jdbc.Driver or com.mysql.cj.jdbc.Driver'
```
Comment (yuanoOo) [inline web/web/src/lib/utils/initial.js:320]: `com.oceanbase.jdbc.Driver` is also available for OceanBase.
```suggestion
        description: 'e.g. com.mysql.jdbc.Driver or com.mysql.cj.jdbc.Driver' or `com.oceanbase.jdbc.Driver`
```

File: web/web/src/lib/utils/initial.js [hunk 3]
```diff
@@ -308,5 +308,33 @@ export const providers = [
...
+      },
+      {
+        key: 'jdbc-url',
+        value: '',
+        required: true,
+        description: 'e.g. jdbc:mysql://localhost:2881'
```
Comment (yuanoOo) [inline web/web/src/lib/utils/initial.js:326]:
```suggestion
        description: 'e.g. jdbc:mysql://localhost:2881' or jdbc:oceanbase://localhost:2881'
```

**Review by yuanoOo — APPROVED** [review]
LGTM

File: web/web/src/app/metalakes/metalake/MetalakeTree.js [hunk 4]
```diff
@@ -72,6 +72,8 @@ const MetalakeTree = props => {
             return 'custom-icons-paimon'
           case 'lakehouse-hudi':
             return 'custom-icons-hudi'
+          case 'oceanbase':
```
Comment (mchades) [inline web/web/src/app/metalakes/metalake/MetalakeTree.js:75]: jdbc-oceanbase

**Review by mchades — APPROVED** [review]
LGTM. Wait for the CI to pass.

**Pull request merged**